### ISSUE TITLE
Compiling against the iOS 14 SDK changes the look of the native APIs, so this library looks extremely weird. 

### ISSUE THREAD

**Issue by SConaway** [react-native-community/datetimepicker#203]
# Bug report

## Summary

Compiling against the iOS 14 SDK changes the look of the native APIs, so this library looks extremely weird. Namely, the pickers have way too much spacing surrounding them.

Library version: 2.4.2

Using the example app to demonstrate, they look like this:

![ezgif com-video-to-gif](https://user-images.githubusercontent.com/20807660/85487791-17545480-b582-11ea-93cf-d04bc5c231ee.gif)

An example of both natively used is in the Calendar app:
![IMG_D21D4E6D455A-1](https://user-images.githubusercontent.com/20807660/85484325-75ca0480-b57b-11ea-9277-a380633683eb.jpeg)

**Comment by jwktje** [comment]
I had this issue too. I read [this article](https://www.swiftjectivec.com/ios-14-notable-uikit-additions/) and looked into the docs and found out that the [preferredDatePickerStyle](https://developer.apple.com/documentation/uikit/uidatepicker/3526124-preferreddatepickerstyle?changes=latest_minor) can be set to [UIDatePickerStyle](https://developer.apple.com/documentation/uikit/uidatepickerstyle?changes=latest_minor) value of `Wheels`, which is the integer 1.

Similar conclusion for a non-react-native project is [mentioned here](https://github.com/xmartlabs/Eureka/issues/2052#issuecomment-648807271).

If there would be a way to add an optional property (iOS only) to this component to override the preferredDatePickerStyle to wheels, that would be great! I would create a PR if i knew enough about actual native development.

**Comment by SConaway** [comment]
Thanks for looking into it. I'll try to make a PR addressing that tomorrow.

**Comment by SConaway** [comment]
PR half-completed. I can't figure out `objective-c`.

**Comment by somoso** [comment]
Just as an added thing, the previously introduced `textColor` prop crashes with these new APIs.

**Comment by gazedash** [comment]
@SConaway can you tell me what does it mean to "Compile against iOS 14 SDK", exactly? Cause I got similar problem, though I didn't change anything. With this package, though on iOS 14 it looks like it is supposed to, but on iOS 13 it is just blank for some reason (only on real device)

**Comment by SConaway** [comment]
When you're using iOS 14, the appearance changes. See the attached screenshots. iOS 14 is on the left; iOS 12 on the right.

<img width="840" alt="Screen Shot 2020-07-11 at 7 22 52 AM" src="https://user-images.githubusercontent.com/20807660/87226260-6b638500-c347-11ea-847b-9bfce7b99a35.png">

<img width="844" alt="Screen Shot 2020-07-11 at 7 22 57 AM" src="https://user-images.githubusercontent.com/20807660/87226264-73232980-c347-11ea-8c5f-8730b472800f.png">

<img width="869" alt="Screen Shot 2020-07-11 at 7 23 02 AM" src="https://user-images.githubusercontent.com/20807660/87226268-7a4a3780-c347-11ea-994a-d681487a4d96.png">

<img width="859" alt="Screen Shot 2020-07-11 at 7 23 06 AM" src="https://user-images.githubusercontent.com/20807660/87226275-80d8af00-c347-11ea-8e80-606fe5cac0a6.png">

**Comment by NeliHarbuzava** [comment]
It seems, that this lib needs maintenance for ios 14.

**Comment by vonovak** [comment]
@NeliHarbuzava I'm expect to get this done in a week or two

**Comment by SConaway** [comment]
Sounds good. Feel free to add more to my PR

**Comment by scarlac** [comment]
Here's a workaround until library gets updated. Add this in `AppDelegate.m` in your `didFinishLaunchingWithOptions`:
```objective-c
- (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(NSDictionary *)launchOptions
{
  // ...
  // Workaround until this lands:
  // https://github.com/react-native-community/datetimepicker/issues/203
  if (@available(iOS 13.4, *)) {
    [UIDatePicker appearance].preferredDatePickerStyle = UIDatePickerStyleWheels;
  }
  // ...
}
```
